Replace deprecated CoreCompat.System.Drawing with new System.Drawing.Common

IGx89 opened the pull request:
System.Drawing.Common is now (as of a month ago) the official solution to using System.Drawing in .NET Core.

CC @axunonb

axunonb commented:
Fully agree, and thanks for submitting the PR.
Great idea and surely deserves a new Nuget release!
Class ```ZedGraph.ColorTranslator``` in file **ColorTranslator.cs** should be **removed** (and dependencies adjusted), because this is part of ```System.Drawing.Common``` now and thus redundant. I had only introduced the class as a missing part for ```CoreCompat.System.Drawing```.

IGx89 commented:
Thanks for pointing that out, @axunonb! I've removed it and have also cleaned up the csproj file a bit due to @discomurray having moved all the WinForms code out of the project. Let me know how it looks now.

IGx89 commented:
Hi @discomurray! Any chance you'd be able to push another preview release to NuGet this week with this PR incorporated?

discomurray commented:
Thanks for the pull request.

IGx89 commented:
Thanks a lot!
